src/matter/cluster/OccupancySensingCluster.ts

```diff
@@ -0,0 +1,63 @@
+/**
+ * @license
+ * Copyright 2022 The node-matter Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Attribute, OptionalAttribute, Cluster, Command, TlvNoArguments, TlvNoResponse } from "./Cluster";
+import { BitFlag, MatterApplicationClusterSpecificationV1_0, TlvBitmap, TlvInt16, TlvUInt16, TlvEnum, TlvField, TlvNullable, TlvObject, TlvSchema, TlvUInt8 } from "@project-chip/matter.js";
+
+/**
+ * Attributes and commands for Occupancy Detection.
+ *
+ * @see {@link MatterApplicationClusterSpecificationV1_0} § 2.7
+ */
+
+
+/** @see {@link MatterApplicationClusterSpecificationV1_0} § 2.7.5.1 */
+export const occupancyBitmap = TlvBitmap(TlvUInt8, {
+    occupied: BitFlag(0),
+})
+
+/** @see {@link MatterApplicationClusterSpecificationV1_0} § 2.7.5.2 */
+export const enum OccupancySensorType {
+    PIR = 0,
+    Ultrasonic = 1,
+    PIRAndUltrasonic = 2,
+    PhysicalContact = 3,
+}
+
+/** @see {@link MatterApplicationClusterSpecificationV1_0} § 2.7.5.3 */
+export const OccupancySensorTypeBitmap = TlvBitmap(TlvUInt8, {
+    PIR: BitFlag(0),
+    ultrasonic: BitFlag(1),
+    physicalContact: BitFlag(2)
+})
+
+export const OccupancySensingCluster = Cluster({
+    id: 0x0406,
+    name: "OccupancySensing",
+    revision: 3,
+
+    /** @see {@link MatterApplicationClusterSpecificationV1_0} § 2.7.5 */
+    attributes: {
+       occupancy: Attribute( 0x0000, TlvUInt8 ),
```

Comment:
Hm ... specs say Constrain "0b0000 000x" ... also based on the description it is in fact a BitMap with only Bit 0 being defined. So I would vcreate a own Bitmap for it with "isOccupied" if the bit is set?

Reply:
Ahh you already have defined that Bitmap ... occupancyBitmap ...just use it ;-)

aka instead of TlvUInt8 write OccupancyBitmap